.github/scripts/ep_review.py

```
@@ -0,0 +1,199 @@
+    if not review:
+        print("No structured review returned — check model response.")
+        print(f"Response: {response.content}")
+        sys.exit(1)
+
+    # Format comment
+    comment = format_comment(review)
+    print("\n" + comment + "\n")
+
+    # Find existing AI review comment to update (idempotent)
+    existing = gh(["api", f"repos/{REPO}/issues/{PR_NUMBER}/comments",
+                   "--jq", f'[.[] | select(.body | startswith("## AI EP Review:"))][0].id // empty'])
```

Ruff (0.15.18)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)